lib/suite.js

```diff
@@ -48,17 +51,12 @@ Suite.create = function (parent, title) {
 };
 
 /**
- * Constructs a new `Suite` instance with the given `title`, `ctx`, and `isRoot`.
- *
- * @public
- * @class
- * @extends EventEmitter
- * @see {@link https://nodejs.org/api/events.html#events_class_eventemitter|EventEmitter}
  * @param {string} title - Suite title.
  * @param {Context} parentContext - Parent context instance.
  * @param {boolean} [isRoot=false] - Whether this is the root suite.
  */
-function Suite(title, parentContext, isRoot) {
+constructor(title, parentContext, isRoot) {
+  super();
```

Comment:
just checking if we need to explicitly call `super` here as well. I might be missing something but I don't think we need it